internal/provisioner/cluster_installation_provisioner.go

```diff
@@ -1083,6 +1083,10 @@ func addSourceRangeWhitelistToAnnotations(annotations *model.IngressAnnotations,
 	}
 
 	annotations.WhitelistSourceRange = allIPRanges
+
+	//update the ConfigurationSnippet
+	updatedAnnotations := model.ConfigureIngressAnnotations(allIPRanges, annotations.ConfigurationSnippet)
+	annotations.ConfigurationSnippet = updatedAnnotations.ConfigurationSnippet
```

Comment:
Do we still need whitelist source ranges to be set now? Will that directive not conflict with our configuration snippets?

Reply:
@nickmisasi Not sure. Looks like they don't conflict, during my testing. But your point seems valid.